converters/fromZigbee.js

```diff
@@ -5686,6 +5691,65 @@ const converters = {
             return payload;
         },
     },
+    silvercrest_smart_led_string: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
+            const value = tuyaGetDataValue(msg.data.datatype, msg.data.data);
+
+            const result = {};
+
+            if (dp === common.TuyaDataPoints.silvercrestSetBrightness) {
+                result.brightness = value;
+            } else if (dp === common.TuyaDataPoints.silvercrestSetColor) {
+                const hString = value.substring(0, 4);
+                const sString = value.substring(4, 8);
+                const bString = value.substring(8, 12);
+
+                const h = parseInt(hString, 16);
+                const s = parseInt(sString, 16);
+                const b = parseInt(bString, 16);
+
+                if (!meta.state.color) {
+                    result.color = {};
+                }
+                result.color.b = (b / 1000) * 255;
+                result.color.h = h;
+                result.color.s = s / 10;
+            } else if (dp === common.TuyaDataPoints.silvercrestSetScene) {
+                const sceneValue = value.substring(0, 2);
+                const sceneString = Object.keys(common.silvercrestEffects).find(
```

Comment:
Can be simplified to `utils.getKeyByValue` (https://github.com/Koenkk/zigbee-herdsman-converters/blob/2198313284afe7b40d139e8bded6ba5974c20c6e/converters/utils.js#L181)

Reply:
Done